.registryrc

```diff
@@ -1 +1 @@
-v4.7.0
+9dd9dd76cc4edca6aa4ce9be3b8a3aa54b19a6a6
```

Comment:
will update once https://github.com/hyperlane-xyz/hyperlane-registry/pull/274 is merged